src/main/java/core/basesyntax/model/User.java

```diff
@@ -0,0 +1,35 @@
+package core.basesyntax.model;
+
+public class User {
+    private String login;
+    private int id;
```

Comment:
let's not have an id for now
in future all our entities will have an id, not only user
```suggestion
    private int age;
```